docs/source/conf.py

```diff
@@ -204,6 +205,9 @@
     "community_links": community_links,
     "language_bindings_links": language_bindings_links,
     "version": version,
+    "date_info": {
+        "paths_to_skip": ["generated/", "index"],
```

Comment:
I guess it doesn't work on generated pages?
Because most of our doc are actually these generated pages like https://docs-preview.pytorch.org/pytorch/pytorch/151602/generated/torch.is_tensor.html#torch.is_tensor and it would be good to have the information for these as well !

Reply:
This feature uses `git log`. There is no git log information for the `generated` pages because they are *generated* on build and not stored in the repo.